library/js/prescription-autocomplete/prescription-autocomplete.js

```diff
@@ -0,0 +1,129 @@
+/*
+ *  @package   OpenEMR
+ *  @link      http://www.open-emr.org
+ *  @author    Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @copyright Copyright (c )2019. Sherwin Gaddis <sherwingaddis@gmail.com>
+ *  @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
+ *
+ *
+ */
+
+
+
+$(document).ready(function () {
+    $("#sendScript").toggle();
+    $('#pharmacy-autocomplete').autocomplete({
+
+        source: function (request, response) {
+            // Fetch data
+            let autocompleteUrl = $("#pharmacyselect").data('autocomplete-url');
```

Comment:
need:
top.restoreSession();

Reply:
done

Reply:
still need top.restoreSession();

Reply:
hi @juggernautsei , looking to add Weno to a site running 5.0.2.1. Grabbed the zip file you've provided at https://omp.openmedpractice.com/dev/weno/ but am not seeing some of the code suggestions here in the files in the zip. Just want to be sure before I bring this into a production environment since they're pretty keen on getting going as you could imagine :)

Reply:
@stephenwaite, the patch that I created is to use Twig. That code should be included here. Brady was speaking of untwigging the code for the official patch. I don't know what this is waiting on. I can check it out and see if there was anything major that I changed. I will get back to you Friday.

Reply:
hi @juggernautsei , ok, thank you.

Reply:
hi @juggernautsei , there are still some unresolved issue on this PR. Let me know if want me to point them out.

Reply:
hi @juggernautsei , it'd be great if you can finish this up with Brady's assistance and then he can help push an untwigged version to 5.0.2 so that when a critical security issue is fixed with the portal we can then push out patch 2. Am a little nervous running docker require on an aws docker production environment, thank you